shared/locales/fi/translation.json

```
@@ -195,6 +195,11 @@
   ],
   "homeHeroGradientSubtext": "Konseptista koodiin me rakennamme ihmislähtöisiä GenAI-kokemuksia.",
   "homeHeroSubtext": "Konseptista koodiin me rakennamme ihmislähtöisiä GenAI-kokemuksia.",
+  "homeHeroSubtextOptions": [
+    "Konseptista koodiin me rakennamme ihmislähtöisiä GenAI-kokemuksia.",
+    "Konseptista koodiin, suunnittelu ja toteutus yhdessä tiimissä.",
```

Biome (2.1.2)

[error] The key homeHeroSubtext was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)